apps/ledger-live-mobile/src/config/urls.tsx

```diff
@@ -164,3 +165,41 @@ export const urls = {
       "https://www.ledger.com/supported-services?utm_source=ledger_live&utm_medium=self_referral&utm_content=discover",
   },
 };
+
+const regex = /{locale}/g;
+
+class UrlsConfig {
+  public locale = "en";
+
+  changeLocale(locale: string) {
+    this.locale = locale;
+  }
+}
+
+export const urlsConfig = new UrlsConfig();
+
+// Iterate through url strings and substitute them by getter function with regex replacing template {locale} by current locale
+const iterateUrls = (obj: typeof urls) => {
+  Object.keys(obj).forEach(key => {
+    // @ts-ignore
+    if (typeof obj[key] === "object" && obj[key] !== null) {
+      // @ts-ignore
+      iterateUrls(obj[key]);
+    }
+
+    // @ts-ignore
+    if (typeof obj[key] === "string") {
+      // @ts-ignore
+      // eslint-disable-next-line no-param-reassign
+      obj["_" + key] = obj[key];
+      Object.defineProperty(obj, key, {
+        get() {
+          // @ts-ignore
```

Comment:
can we avoid adding more ts-ignore in our codebase? 🙏  not sure what the error was but we may have to find a better algorithm. 

Also, I'm not a huge fan of the usage of `get()` here. It will not help performance to have a lot of these kind of code.
I totally understand that i was meant not to be breaking change tho. But, we are making `urls` dynamic, so in a way it's a new feature, it is ok to make it explicit 🤔  I guess that means changing all parts where we get that URL, but I feel it's for the better good.

There are probably different ways to do this, one way could be a function `getURL("key", lang)`. We can also cache per lang / refresh a map per lang. anything that could try to avoid doing this at runtime 🙏